README.md

```diff
@@ -139,26 +139,129 @@ Dedicated servers hosted on Steam are usually required to be running the *latest
 
 Clean images are tagged by `<version>`. Layered images are tagged by `<version>-layered`.
 
-### Image Size
+### Image size
 
 Image sizes shown above or on Docker Hub correspond to an image's *compressed* size. Actual sizes vary, but are approximately **2x** larger after pulling an image.
 
-### Update Duration
+### Update duration
 
 From the moment Valve issues an update, the time taken before a game's images are built and available for pulling largely depends on the size of the game. For instance, layered and clean images take over **15** and **40 minutes** respectively for `Counter-Strike: Global Offensive`, but under **5 minutes** each for `Counter-Strike 1.6`.
 
 While the use of build cache can help drastically reduce update durations, it cannot be utilized as the game images are built for public use, purposefully done so using public machines.
 
-## Build history
+### Build history
 
 The project uses multiple CI services for its build jobs. You can find the history of past build jobs by clicking on their corresponding build status badges.
+
+## Usage
+
+### Docker
+
+The project assumes knowledge concerning the [`docker`](https://docs.docker.com/) runtime. Instructions on customization and orchestration of containerized game server instances are outside the scope the project.
+
+#### Entrypoint and CMD
+
+Currently, the default **ENTRYPOINT** for all game images is [`"bash", "-c"`](build/Dockerfile#L72), and the **CMD** is [`""`](build/Dockerfile#L73). This makes it convenient especially in development environments in that the game's command line can simply be appended as the final argument of the `docker run` command.
+
+Each of the values can be overridden at runtime which is well supported by container orchestration tools such as [Kubernetes](https://kubernetes.io/) and  [Docker Swarm Mode](https://docs.docker.com/engine/swarm/), and the standalone tool, [Docker Compose](https://docs.docker.com/compose/). Alternatively, they can be modified as part of the build steps in custom images.
+
+#### Workdir
+
+The default working directory for all the images is [`/server`](build/Dockerfile#L70) within which all of a game's dedicated server files reside.
+
+#### Command line
+
+The following are some examples of how the game servers can be run:
+
+```shell
+# Counter-Strike: Global Offensive
+docker run -it -p 27015:27015/udp sourceservers/csgo:latest 'srcds_linux -game csgo -port 27015 +game_type 0 +game_mode 1 +mapgroup mg_active +map de_dust2'
+
+# Counter-Strike 1.6
+docker run -it -p 27016:27016/udp goldsourceservers/cstrike:latest 'hlds_linux -game cstrike +port 27016 +maxplayers 10 +map de_dust2'
```

Comment:
```suggestion
docker run -it -p 27016:27016/udp goldsourceservers/cstrike:latest hlds_linux -game cstrike +port 27016 +maxplayers 10 +map de_dust2
```

Reply:
The game command line needs to be quoted as a string as a final argument to `docker run` because of the entrypoint `"bash" ,"-c"`. Try it out unquoted and see that the server fails to start correctly because `-game cstrike` isn't passed correctly.

Reply:
That's correct. Missed that.